dubbo-common/src/main/java/org/apache/dubbo/common/constants/CommonConstants.java

```diff
@@ -183,6 +183,8 @@
 
     String GROUP_KEY = "group";
 
+    String NACOS_GROUP_KEY = "nacos.group";
```

Comment:
It would be better if we move this constant to dubbo-registry-nacos module

Reply:
good idea. i will modify it.